rfcs/127-add-serverinternel-and-requestthrottled-errors.md

```diff
@@ -0,0 +1,41 @@
+- Author: xxchan <xxchan22f@gmail.com>
+- Start Date: 2021-06-24
+- RFC PR: [beyondstorage/specs#127](https://github.com/beyondstorage/specs/issues/127)
+- Tracking Issue: [beyondstorage/go-storage#0](https://github.com/beyondstorage/go-storage/issues/0)
+
+# GSP-127: Add ServiceInternel and RequestThrottled Errors
+
+## Background
+
+These are common errors, especially in HTTP-based services, but also apply to RPC-based services. And they are retryable errors. If we provide them, users can retry only them instead of all errors.
+
+## Proposal
+
+I propose to add the following global error codes:
```

Comment:
Is it necessary to add error types to point out the error code or error message?

Reply:
I think we don't need to do so now. `fmt.Errorf("%w: %v", ErrServiceInternal, err) is enough.